### PR TITLE
Fix API doc parameter placeholder: last

### PR DESCRIPTION
**Pull request by Jamstah**
Last is not an integer, but is the last returned value to start the
pagination from.

Signed-off-by: James Hewitt <james.hewitt@uk.ibm.com>

Recreate of #3672 and #3673 with a signed DCO.

### CODE REVIEW

**Review by milosgajdos — APPROVED** [review]

**Comment by Jamstah** [comment]
@thaJeztah as you seem active right now, could I get a second review to close this simple doc fix?

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**
